package.json

```diff
@@ -36,16 +36,12 @@
     "@babel/preset-env": "^7.17.10",
     "@babel/preset-typescript": "^7.16.7",
     "@changesets/cli": "^2.24.4",
-    "@metaplex-foundation/amman": "^0.12.0",
-    "@metaplex-foundation/amman-client": "^0.2.2",
```

Comment:
These are used by the `amman:start` script.

Reply:
it was duplicated. it is still in the package json